workspaces/bulk-import/plugins/bulk-import/src/components/AddRepositories/AddRepositories.tsx

```diff
@@ -34,23 +36,11 @@ import { PreviewFileSidebar } from '../PreviewFile/PreviewFileSidebar';
 import { AddRepositoriesFormFooter } from './AddRepositoriesFormFooter';
 import { AddRepositoriesTable } from './AddRepositoriesTable';
 
-const useStyles = makeStyles(theme => ({
+const useStyles = makeStyles(() => ({
   body: {
     marginBottom: '50px',
     padding: '24px',
   },
-  approvalTool: {
-    display: 'flex',
-    flexDirection: 'row',
-    justifyContent: 'left',
-    alignItems: 'center',
-    paddingTop: '24px',
-    paddingBottom: '24px',
-    paddingLeft: '16px',
-    backgroundColor: theme.palette.background.paper,
-    borderBottomStyle: 'groove',
-    border: theme.palette.divider,
-  },
```

Comment:
Can you comment this for now instead of removing. We would need this once the `ServiceNow` approval tool is incorporated in the plugin

Reply:
@its-mitesh-kumar please add comment describing why this code is commented out.